src/components/layout/NavBar.less

```diff
@@ -0,0 +1,58 @@
+.nav-bar {
+  width: 100%;
+  background-color: white;
+  border-bottom: 1px solid #d1d1d1;
+}
+
+.nav-bar__nav--left {
+  float: left;
+  display: inline-flex;
+}
+
+.nav-bar__nav--right {
+  float: right;
+  color: white;
+  overflow: hidden;
+}
+
+.nav-bar__header {
+  overflow: hidden;
+  margin: auto;
+  width: 1220px;
+}
+
+.nav-bar__link {
```

Comment:
I think this is missing the :hover and :active properties

Reply:
:hover and :active is already defined in `NavButton.less`, previous code duplicately defined them. already tested in my local and hover/active effects work fine.